### PR TITLE
Adding Job, New TF2.7 conda and ads v2.5.6 

### PR DESCRIPTION
**Pull request by jrgauthier01**
Updates to the MLSummit lab. This is probably the most comprehensive E2E lab for the service.

### CODE REVIEW

File: labs/MLSummit21/lab-0-tenancy-setup.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ In this first lab, you will:
 
 Sign up [here](https://www.oracle.com/cloud/free/). 
 
+:exclamation: :exclamation: :exclamation: **If you already have a tenancy make sure that you have not exhausted the Free Trial credits**. If you have exhausted the credits or your tenancy is older than 30 days, you will only have access to "Always Free" Services. **OCI Data Science is not yet among the "Always Free" offerings.** You will have to convert your tenancy to a paid tenancy or use a different tenancy.   
+
 ## **STEP 2:** Run the Data Science Stack Template 
 
 We have created a Terraform script that can be executed throught the Resource Manager Stack resource. This Terraform script creates the basic user groups, policies, dynamic groups, networking (VCN and subnets) required to create projects and notebook sessions. The Stack also allows you to optionally launch a notebook session after teh setup is completed. We recommend that you create the notebook session.
```
Comment (wprichard) [inline labs/MLSummit21/lab-0-tenancy-setup.md:22]: typos: throught, teh

File: labs/MLSummit21/lab-45-training-job.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Automate your typical MLOps or CI/CD pipeline.
+* Execute batches or workloads triggered by events or actions.
+* Batch, mini batch, or distributed batch job inference.
+
+After the steps are completed, you can automate the process of data exploration, model training, deploying and testing using jobs. A single change in the data preparation or model training, experiments with hyperparameter tunings could be run as Job and independently tested.
+
+Jobs are two parts, a job and a job run:
+
+### Job
+A job is template that describes the task. It contains elements like the job artifact that is immutable and can't be modified after it's uploaded to a job. Also, the job contains information about the Compute shapes the job runs on, logging options, block storage, and other options. You can add environment variables or CLI arguments to jobs to be unique or similar for all your future job runs. You can override these variables and arguments in job runs.
```
Comment (wprichard) [inline labs/MLSummit21/lab-45-training-job.md:20]: Missing "a": A job is a template...

**Review by wprichard — APPROVED** [review]
Looks good. I spotted a couple of minor markdown typos and made comments.

File: labs/MLSummit21/lab-45-training-job.md (hunk 2 above)
Comment (jrgauthier01) [reply labs/MLSummit21/lab-45-training-job.md]: done thanks.

File: labs/MLSummit21/lab-0-tenancy-setup.md (hunk 1 above)
Comment (jrgauthier01) [reply labs/MLSummit21/lab-0-tenancy-setup.md]: Thanks. done.

**Pull request merged**